app/helpers/application_helper.rb

```diff
@@ -64,4 +64,13 @@ def claim_decision_deadline_in_weeks
   def done_page_url
     "https://www.gov.uk/done/claim-additional-teaching-payment"
   end
+
+  def information_provided_further_details_with_link(policy_str)
+    policy = policy_str.present? ? Policies.constantize(policy_str) : current_claim.policy
+
+    text = I18n.t("#{policy.locale_key}.information_provided_further_details_link_text")
+    link = link_to(text, policy.payment_and_deductions_info_url, class: "govuk-link govuk-link--no-visited-state", target: "_blank")
+
+    I18n.t("#{policy.locale_key}.information_provided_further_details", link: link)&.html_safe
+  end
```

Comment:
A part of me is tempted to add a test for this 😁 but I bet this helper doesn't have unit tests?

Reply:
There are some unit tests for helpers, but that wouldn't be a reason not to add one for a new method :)